lib/lambda.js

```diff
@@ -50,36 +22,43 @@ module.exports = function elasticApmAwsLambda (agent) {
   }
 
   function wrapContext (trans, payload, context) {
-    const succeed = wrapLambdaCallback(trans, payload, context, (_, res) => context.succeed(res))
-    const done = wrapLambdaCallback(trans, payload, context, (err, res) => context.done(err, res))
-    const fail = wrapLambdaCallback(trans, payload, context, (err) => context.fail(err))
+    shimmer.wrap(context, 'succeed', (succeed) => {
+      return function wrappedSucceed (result) {
+        var bound = succeed.bind(this, result)
```

Comment:
We should probably just keep using `const` here (and elsewhere in the file)

Reply:
Good catch. 😅

Reply:
Fixed.